app/views/challenges/_overview.html.erb

```diff
@@ -16,7 +16,7 @@
           </div>
           <div class="row row-cta">
             <div class='text-center center-block'>
-              <%= link_to 'Join the Challenge', challenge_active_stage_path(challenge), class: 'btn btn-lg btn-rounded btn-longer btn-info half-margin-bottom btn-cta' %>
+              <%= link_to 'Join the Challenge', challenge_experience_stage_path(challenge, challenge.experience_stage), class: 'btn btn-lg btn-rounded btn-longer btn-info half-margin-bottom btn-cta' %>
```

Comment:
@stedmanblake I'd put this back as it was previously. The `active_stage_path` gets set by setting the 'active_stage' column on the Challenge. Now that it's set to challenge, it will appropriately be stage 1 (and then will transition automatically everywhere as we go through each stage.